pkgs/games/quake3/quake3e/default.nix

```
@@ -0,0 +1,36 @@
+{ stdenv, curl, libGL, libX11, libXxf86dga, alsaLib, libXrandr, libXxf86vm, libXext, fetchFromGitHub }:
+stdenv.mkDerivation rec {
```

please add a space :)
```suggestion

stdenv.mkDerivation rec {
```